### DISCUSSION TITLE
Achievments

### DISCUSSION THREAD

**Discussion by ArturRSA19** [community/community#55551] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Thanks for creating GitHub Achievments. They boost the devs to develop more and be more dedicated!

**Comment by RevertIT** [comment]
Start contributing more! 

You gotta catch them achievementz!

**Comment by YasinDehfuli** [comment]
## This course can help everyone who want to [ get GitHub achievements](https://github.com/4xmen/Get-Github-Achievements).

### how to get : 
#### https://github.com/4xmen/Get-Github-Achievements
